src/main/java/com/sprint/mission/discodeit/repository/file/BasicFile.java

```diff
@@ -0,0 +1,65 @@
+package com.sprint.mission.discodeit.repository.file;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sprint.mission.discodeit.entity.User;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BasicFile {
+
+  private Path fileDirectory;
```

Comment:
생성자에서 초기화한다면 final을 붙여도 될 것 같아요.  
```suggestion
  private final Path fileDirectory;
```